### PR TITLE
Add support for Color attribute loading from FBX model

### PR DESCRIPTION
**Pull request by samcake**
- load the attribute LayerElementColor from the FBX geometry model and bring it in the model::Geometry used for rendering

- Tweak the model.slv and model.slf to support the color info comming from the mesh

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: f0efade06d48e6d0fe2d3612744dcb0d26c3933b available at: https://gist.github.com/236ea914b4ce76084bd4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4607.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4607.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ad9129b552e7d86f5b06a9f7a3801ca61e7fc053 available at: https://gist.github.com/236ea914b4ce76084bd4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4607.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4607.dmg

**Comment by samcake** [comment]
Ready for review

**Comment by jherico** [comment]
@samcake can you provide a link to an example FBX with which to test this?

**Comment by samcake** [comment]
YEp :)

http://samcake.github.io/models/terrain/Tree_BlossomB__cv1.fbx

File: libraries/render-utils/src/model_lightmap.slv [hunk 1]
```diff
@@ -28,9 +28,12 @@ varying vec4 normal;
 // the interpolated texcoord1
 varying vec2 interpolatedTexcoord1;
 
+varying vec3 color;
+
+
 void main(void) {
     // pass along the diffuse color
-    gl_FrontColor = gl_Color;
+    color = gl_Color.xyz;
```
Comment (jherico) [inline libraries/render-utils/src/model_lightmap.slv]: no big deal, but these should probably be expressed as `gl_Color.rgb`.

**Comment by jherico** [comment]
Tested with the old code and the new on my windows desktop.  Old code produces a white tree, while the new code reflects the colors in the model producing a nice cherry tree.

File: libraries/render-utils/src/model_lightmap.slv (hunk 1 above)
Comment (samcake) [inline libraries/render-utils/src/model_lightmap.slv]: Don t get me started on this one! :)
Ok maybe

**Pull request merged**